contrib-02-lhcb-openstack/Readme.md

```diff
@@ -0,0 +1,192 @@
+# LHCbSoftware on Openstack
+
+## Motivation
+
+The easiest way to to work in an LHCb Software Evironment is by logging in to an `lxplus` instance.
+From there one also has access to the [`AFS` file storage](http://information-technology.web.cern.ch/services/afs-service).
+
+However, working on `lxplus` comes with a few significant limitations:
+ * No super user access. This makes it hard to install new software.
+ * Shared resources with other users. Others might start resource-hungry processes on the same machine, therefore extending execution time and making it unpredicatable.
+
+Using a virtual machine on [`openstack.cern.ch`](https://openstack.cern.ch) solves these issues. It is possible to disposition a machine with up to 10 vCPUs and up to 20 GB memory. While vCPUs are not the same as actual, physical CPUs, this comes with with a significant improvement in performance and performance reliablity.
```

Comment:
@alexpearce You are right, I will address this.
@apuignav What kind of comment do you mean?